modules/enabling-entra-workload-id-existing-cluster.adoc

```diff
@@ -0,0 +1,246 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-entra-workload-id-existing-cluster"]
+= Enabling {entra-first} on an existing cluster
+
+If you did not configure your {azure-first} {product-title} cluster to use {entra-first} during installation, you can enable this authentication method on an existing cluster.
+
+[IMPORTANT]
+====
+This process is disruptive and takes a significant amount of time.
+Before proceeding, review the following steps and ensure that you accept the time requirement.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster on {azure-first}.
+* You have access to the cluster using an account with `cluster-admin` permissions.
+* You have installed the {oc-first}.
+* You have extracted and prepared the Cloud Credential Operator utility (`ccoctl`) binary.
+* You have access to your {azure-short} account by using the {azure-short} CLI (`az`).
+
+.Procedure
+
+. Create an output directory for the manifests that the `ccoctl` utility generates.
+This procedure uses `./output_dir` as an example.
+
+. Extract the `ServiceAccount` public signing key for the cluster to the output directory by running the following command:
++
+[source,terminal]
+----
+$ oc get configmap \
+  --namespace openshift-kube-apiserver bound-sa-token-signing-certs \
+  --output 'go-template={{index .data "service-account-001.pub"}}' > ./output_dir/serviceaccount-signer.public <1>
+----
+<1> This procedure uses a file named `serviceaccount-signer.public` as an example.
+
+. Use the extracted `ServiceAccount` public signing key to create an OpenID Connect (OIDC) issuer and {azure-short} blob storage container with OIDC configuration files by running the following command:
++
+[source,terminal]
+----
+$ ./ccoctl azure create-oidc-issuer \
+  --name <azure_infra_name> \// <1>
+  --output-dir output_dir \
+  --region <azure_region> \// <2>
+  --subscription-id <azure_subscription_id> \// <3>
+  --tenant-id <azure_tenant_id> \
+  --public-key-file output_dir/serviceaccount-signer.public <4>
+----
+<1> The value of the `name` parameter is used to create a matching Azure resource group.
+To use an existing Azure resource group, specify the `--oidc-resource-group-name` parameter.
+#Do they use both `name` and `oidc-resource-group-name` in that case? If so, can they specify different values?#
+<2> Specify the region of the existing cluster.
+<3> Specify the subscription ID of the existing cluster.
+<4> Specify the file that contains the `ServiceAccount` public signing key for the cluster.
+
```

Comment:
After executing `ccoctl azure create-oidc-issuer`, the configuration file for the Azure pod identity webhook, `azure-ad-pod-identity-webhook-config.yaml`, will be generated in the `output_dir/manifests` directory. 
Next, add a step to verify that the `azure-ad-pod-identity-webhook-config.yaml` file was successfully createdby running the command ll output_dir/manifests.
Example:
```
ll output_dir/manifests
total 8
-rw-------. 1 cloud-user cloud-user 193 May 22 02:29 azure-ad-pod-identity-webhook-config.yaml
-rw-------. 1 cloud-user cloud-user 165 May 22 02:29 cluster-authentication-02-config.yaml
```